src/utils/config.ts

```diff
@@ -3,8 +3,8 @@ import * as fs from 'fs';
 import { Logger } from './logger';
 import Axios from 'axios';
 
-const ConfigurationKey = 'planner.azure-devops';
-const SnippetsConfigurationKey = 'planner.azure-devops.snippets';
+const ConfigurationKey  = 'planner.azure-devops';
+const SnippetsConfigurationKey  = 'planner.azure-devops.snippets';

```

Comment:
One extra space before the equality sign :) (both lines)

Reply:
Darn, you've got an eagle eye. I'm a bit hesistant to use my formatter locally as it is configured to use spaces and it seems that your project is using tabs. hence my suggestion for the editorconfig